ananas/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,109 @@
+<resources>
+    <string name="iamutkarshtiwari_github_io_ananas_library_name">Photo Editor</string>
```

Comment:
This doesn't seem like a french locale. Could you please revert this?

Reply:
French translation added with Arabic translation support.